cypress/integration/send_funds.spec.js

```diff
@@ -0,0 +1,186 @@
+  beforeEach(() => {
+    cy.visit("public/index.html");
+  });
+  it("opens the modal from the user wallet", () => {
+    // user wallet opens modal
+    cy.pick("Wallet").click();
+    cy.pick("send-receive").should("exist");
+    cy.pick("send-tab").click();
+    cy.pick("send").should("exist");
+    cy.pick("recipient-input").type(
+      "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu"
+    );
+    cy.pick("amount-input").type("123");
+    cy.pick("send-transaction-button").click();
+    cy.pick("send-funds-modal").should("exist");
+  });
+  it("opens the modal from the org wallet", () => {
+    // org wallet opens modal
+    cy.pick("sidebar", "org-coolorg").click();
+    cy.pick("Wallet").click();
+    cy.pick("send-receive").should("exist");
+    cy.pick("send-tab").click();
+    cy.pick("send").should("exist");
+    cy.pick("recipient-input").type(
+      "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu"
+    );
+    cy.pick("amount-input").type("123");
+    cy.pick("send-transaction-button").click();
+    cy.pick("send-funds-modal").should("exist");
+  });
+  it("closes the modal by clicking X", () => {
+    cy.pick("Wallet").click();
+    cy.pick("send-receive").should("exist");
+    cy.pick("send-tab").click();
+    cy.pick("send").should("exist");
+    cy.pick("recipient-input").type(
+      "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu"
+    );
+    cy.pick("amount-input").type("123");
+    cy.pick("send-transaction-button").click();
+    cy.pick("send-funds-modal").should("exist");
+    cy.pick("modal-close-button").should("exist");
+    cy.pick("modal-close-button").click();
+    cy.pick("Wallet").should("exist");
+  });
+  it("closes the modal by pressing esc", () => {
+    cy.pick("Wallet").click();
+    cy.pick("send-receive").should("exist");
+    cy.pick("send-tab").click();
+    cy.pick("send").should("exist");
+    cy.pick("recipient-input").type(
+      "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu"
+    );
+    cy.pick("amount-input").type("123");
+    cy.pick("send-transaction-button").click();
+    cy.pick("send-funds-modal").should("exist");
+    cy.get("body").type("{esc}");
+    cy.pick("Wallet").should("exist");
+  });
+  it("is able to go to review step and back when everything is valid", () => {
+    cy.pick("Wallet").click();
+    cy.pick("send-receive").should("exist");
+    cy.pick("send-tab").click();
+    cy.pick("send").should("exist");
+    cy.pick("recipient-input").type(
+      "5DjHdsNAL7L7UynCDUscMsbnYd1DW4DxqJrA8wX4aS3Moyz3"
+    );
+    cy.pick("amount-input").type("123");
+    cy.pick("send-transaction-button").click();
+    cy.pick("send-funds-modal").should("exist");
+    cy.pick("page", "preperation-step").should("exist");
+    cy.pick("review-transfer-button").should("exist");
+    cy.pick("review-transfer-button").click();
+    cy.pick("page", "review-step").should("exist");
+    cy.pick("back-to-review-button").should("exist");
+    cy.pick("back-to-review-button").click();
```

Comment:
We should try to remove the duplication of checks... `cy.pick("back-to-review-button").click();` should be enough, as it also checks if it exists.

Also applies to other areas of the tests

Reply:
Done